vng_api_common/utils.py

```diff
@@ -90,3 +99,15 @@ def get_uuid_from_path(path: str) -> str:
     uuid.UUID(uuid_str)
 
     return uuid_str
+
+
+def request_object_attribute(url: str, attribute: str, resource: Union[str, None] = None) -> str:
+    Client = import_string(settings.ZDS_CLIENT_CLASS)
+    client = Client.from_url(url)
+    client.auth = APICredential.get_auth(url)
+    try:
+        result = client.retrieve(resource, url=url)[attribute]
+    except (ClientError, KeyError) as exc:
+        logger.warning(f"{attribute} was retrieved from {url} with an {exc.__class__.__name__}: {exc}")
```

Comment:
Don't use `f` strings for logging, use old-style logging format: `logger.warning("some %s warning", 'some_arg', exc_info=1)`

1. Sentry aggregates based on the template string, before interpolation
2. Depending on the logging config, it may be below the treshold to actually be logged -> Python Logging module will then not even interpolate the strings
3. If the string cannot be interpolated, you'll get a hard crash, which doesn't happen with the logging approach.

Reply:
Oh, thanks! I didn't know. `logger.warning` format is changed